1652701453890	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1652702155129	Sync.Service	DEBUG	User-Agent: Firefox/100.0 (Intel Mac OS X 10.15) FxSync/1.102.0.20220428192727.desktop
1652702155129	Sync.Service	INFO	Starting sync at 2022-05-16 13:55:55 in browser session ij53ff9hD5M_
1652702155129	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1652702155131	Sync.Status	INFO	Resetting Status.
1652702155131	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1652702155131	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1652702155131	FirefoxAccounts	DEBUG	getOAuthToken enter
1652702155131	FirefoxAccounts	TRACE	getCachedToken returning cached token
1652702155131	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1652702155132	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1652702155144	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/195039599/info/collections failed: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
1652702155144	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
1652702155145	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1652702155147	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
